daemon/execdriver/native/create.go

```diff
@@ -222,3 +227,17 @@ func (d *driver) setupLabels(container *libcontainer.Config, c *execdriver.Comma
 
 	return nil
 }
+
+func (d *driver) setupAdditionalGroups(container *libcontainer.Config, c *execdriver.Command) error {
+	container.AdditionalGroups = make([]int, len(c.GroupAdd))
+
+	for i, g := range c.GroupAdd {
+		group, err := strconv.Atoi(g)
```

Comment:
I think they would have to be from the host's groups file. They could be picked from the container but different containers could have same group names mapped to different gids and that kind of defeats the purpose here.